integration/RUNLOCAL

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# To use this script, drop a bzImage here and away you go.
+# Some examples:
+# ./RUNLOCAL
+# ./RUNLOCAL -v  -test.run TestTczclient
+UROOT_QEMU="qemu-system-x86_64" UROOT_KERNEL=bzImage go test "$@"
```

Comment:
see this usage? There's nothing set in .bashrc. It's set on the command line.

Reply:
And also note with this usage it's not pointing to $HOME/bin/qemu-system-x86_64

Reply:
Think about why this works. Why does it work without the full location?

Reply:
UROOT_QEMU uses the $PATH. Since qemu-system-x86_64 is stored inside of directory in $PATH, it knows exactly where to locate it. and this will work as long as you drop a bzImage (image of the kernel) inside because then it has the kernel and the qemu-system-x86_64 to run locally.